src/app-constants.js

```diff
@@ -8,59 +8,61 @@ import * as dotenv from 'dotenv'
 dotenv.config({ path: '../.env' })
 
 const requiredEnvVars = [
-  'NODE_ENV',
-  'SERVER_URL',
-  'LOGOS_ORIGIN',
+  'ADMINS',
+  'BREACH_RESOLUTION_ENABLED',
   'COOKIE_SECRET',
   'CSRF_SECRET',
-  'SESSION_DURATION_HOURS',
-  'SMTP_URL',
+  'DATABASE_URL',
+  'DELETE_UNVERIFIED_SUBSCRIBERS_TIMER',
   'EMAIL_FROM',
-  'SES_CONFIG_SET',
-  'SES_NOTIFICATION_LOG_ONLY',
-  'BREACH_RESOLUTION_ENABLED',
-  'FXA_ENABLED',
-  'FXA_SETTINGS_URL',
+  'EXPERIMENT_ACTIVE',
+  'FX_REMOTE_SETTINGS_WRITER_PASS',
   'FX_REMOTE_SETTINGS_WRITER_SERVER',
   'FX_REMOTE_SETTINGS_WRITER_USER',
-  'FX_REMOTE_SETTINGS_WRITER_PASS',
-  'MOZLOG_FMT',
-  'MOZLOG_LEVEL',
-  'OAUTH_AUTHORIZATION_URI',
-  'OAUTH_TOKEN_URI',
-  'OAUTH_PROFILE_URI',
-  'OAUTH_CLIENT_ID',
-  'OAUTH_CLIENT_SECRET',
+  'FXA_ENABLED',
+  'FXA_SETTINGS_URL',
+  'HIBP_API_ROOT',
   'HIBP_KANON_API_ROOT',
   'HIBP_KANON_API_TOKEN',
-  'HIBP_API_ROOT',
+  'HIBP_NOTIFY_TOKEN',
   'HIBP_RELOAD_BREACHES_TIMER',
   'HIBP_THROTTLE_DELAY',
   'HIBP_THROTTLE_MAX_TRIES',
-  'HIBP_NOTIFY_TOKEN',
-  'DATABASE_URL',
+  'LOGOS_ORIGIN',
+  'MAX_NUM_ADDRESSES',
+  'MOZLOG_FMT',
+  'MOZLOG_LEVEL',
+  'NODE_ENV',
+  'OAUTH_AUTHORIZATION_URI',
+  'OAUTH_CLIENT_ID',
+  'OAUTH_CLIENT_SECRET',
+  'OAUTH_PROFILE_URI',
+  'OAUTH_TOKEN_URI',
   'PAGE_TOKEN_TIMER',
   'PRODUCT_PROMOS_ENABLED',
   'REDIS_URL',
   'SENTRY_DSN',
-  'DELETE_UNVERIFIED_SUBSCRIBERS_TIMER',
-  'EXPERIMENT_ACTIVE',
-  'MAX_NUM_ADDRESSES',
+  'SERVER_URL',
+  'SES_CONFIG_SET',
+  'SES_NOTIFICATION_LOG_ONLY',
+  'SESSION_DURATION_HOURS',
+  'SMTP_URL',
   'SUPPORTED_LOCALES',
   'GA4_MEASUREMENT_ID'
 ]
 
 const optionalEnvVars = [
-  'PORT',
-  'RECRUITMENT_BANNER_LINK',
-  'RECRUITMENT_BANNER_TEXT',
-  'GEOIP_GEOLITE2_PATH',
+  'EDUCATION_VIDEO_URL_RELAY',
+  'EMAIL_TEST_RECIPIENT',
   'GEOIP_GEOLITE2_CITY_FILENAME',
   'GEOIP_GEOLITE2_COUNTRY_FILENAME',
-  'VPN_PROMO_BLOCKED_LOCALES',
-  'EDUCATION_VIDEO_URL_RELAY',
+  'GEOIP_GEOLITE2_PATH',
   'MONITOR_V2',
-  'SENTRY_DSN_LEGACY'
+  'PORT',
+  'RECRUITMENT_BANNER_LINK',
+  'RECRUITMENT_BANNER_TEXT',
+  'SENTRY_DSN_LEGACY',
+  'VPN_PROMO_BLOCKED_LOCALES'
```

Comment:
praise: I love your alphabetizing skills!

Reply:
I must admit that there might be a plugin helping me out here.